### PR TITLE
LPS-190009 Replace aui:button with clay:button in layout-admin-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-190009)

## Steps

1. Go to Publishing > Staging 
2. Enable Staging > Local Live > Enable Pages Variation
3. Go to pages > Create a Widget Page
4. Go to the Widget Page view
5. Next to Publish to Live button > Click on the kebab > Site Pages Variation
6. Add a site page variation
7. Select the page variation created in page view, it should be selected when it's created
8. Click on configuration icon upper right
9. See the buttons
Screenshot 2023-08-07 at 12 42 26

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/edit_layout.jsp [hunk 1]
```diff
@@ -63,9 +63,17 @@ renderResponse.setTitle(layoutsAdminDisplayContext.getConfigurationTitle(selLayo
...
+				label='<%= LanguageUtil.format(request, "enable-in-x", HtmlUtil.escape(layoutSetBranchName), false) %>'
+			/>
+
+			<clay:button
+				displayType="secondary"
+				id='<%= liferayPortletResponse.getNamespace() + "deleteLayoutButton" %>'
+				label="delete-in-all-pages-variations"
+			/>
 
 			<script>
```
Comment (victorg1991) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/edit_layout.jsp]: What do you think about moving this script to a js file? :innocent: or at least to a <aui:script>

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
@victorg1991 updated! 😄

**Comment by victorg1991** [comment]
Nice :) LGTM

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**